### ISSUE TITLE
about draft-ietf-tsvwg-careful-resume

### ISSUE THREAD

**Issue by marcblanchet** [huitema/quic-in-space#4]
Should we talk about draft-ietf-tsvwg-careful-resume? that could be useful?

**Comment by huitema** [comment]
Yes. That should come in the recommendation parts. The draft should cite all these efforts.
